integration/deployment/Dockerfile

```diff
@@ -0,0 +1,7 @@
+#target: danm-deployer
+FROM alpine:3.8
+MAINTAINER Nandor Galambosi <nandor.galambosi@nokia-bell-labs.com>
+COPY .cache/kubectl /opt/bin/
+COPY bin/* /opt/bin/
```

Comment:
svcwatcher and netwatcher binaries you don't need to install into /opt/bin on the target
I guess they are currently added to the installer container, unnecessarily

Reply:
This container is used only as an initContainer. And initContainer will install all binaries to kubernetes nodes local filesystem.

Reply:
yep, and my comment is that svcwatcher and netwatcher binaries are not required to be installed on the local filesystem
these binaries are only needed within their own container image, which we build separately

only danm, and fakeipam are required to be copied to the filesystem for Kubelet

Reply:
Ok, only danm, and fakeipam will be copied